com/truckmuncher/api/trucks.proto

```diff
@@ -30,11 +30,20 @@ service TruckService {
     /**
      * Modify the serving mode for a truck.
      *
-     * This call requires Vendor authorization
+     * This call is an alias for reportServingMode
+     * This call is deprecated and should no longer be used by clients. It will be removed in the near future.
+     * This call requires the user to be logged in
      */
     rpc modifyServingMode (ServingModeRequest) returns (ServingModeResponse);
```

Comment:
Ah ok, I think I tried to ask that the first time but I wasn't very clear. That makes sense then to keep everything like you have it.